### ISSUE TITLE
Change all hardcoded links to 18f.gsa.gov pages

### ISSUE THREAD

**Issue by beechnut** [18F/18f.gsa.gov#3860]
If you search the content folder for `//18f.gsa.gov/`, there are about 750 results.

At some point, we should change all of these links to the 11ty style of linking: `{{ the path | url }}`

**Problem**: In the absence of analytics, don't know if any of these links are broken, or how many, because we only check internal links, because [checking external links is hard](https://github.com/18F/TLC-crew/issues/354).

**Benefit**: If we change the linking from hardcoded to `{{ }}`, all of these links become internal links. (Right now, they're considered external when you're working locally, on `localhost`.) If these links become internal, they become check-able, improving the integrity of the site.

It's debatable whether this is in or out of scope for the migration, but since it's a non-trivial content change that **doesn't** block migration, I'm going to set it as out-of-scope for now.

```[tasklist]
### Tasks
- [ ] `content/` folder contains zero instances of `//18f.gsa.gov`
- [ ] Bonus points: a `npm run test:something` script checks each PR to make sure it stays that way
```

**Comment by bpdesigns** [comment]
straightfoward and useful task

**Comment by beechnut** [comment]
@bpdesigns This is a post-migration task. This shouldn't be prioritized until after #3839 is complete.

**Comment by bpdesigns** [comment]
Thanks @beechnut I'll add this to blocked/waiting for now
